### PR TITLE
WIP Sample pv doc and cleanup

### PR DESCRIPTION
**Pull request by doxiao**
1) Initial README.md for create-pv-pvc.sh script.
2) Cleanup in the scripts and Java integration test code.

@anpanigr @maggiehe00 @vanajamukkara

### CODE REVIEW

File: kubernetes/samples/scripts/create-weblogic-domain-pv-pvc/README.md [hunk 1]
```diff
@@ -0,0 +1,227 @@
+# Sample Persistent Volume and Persistent Volume Claim
+
+The sample scripts demonstrate the creation of a Kubernetes persistent volume (PV) and persistent volume claim (PVC), which can then be used in a domain custom resource for the domain home or logs.
+
+## Prerequisites
+
+The following pre-requisites must be handled prior to running the create script:
```
Comment (rosemarymarano) [inline kubernetes/samples/scripts/create-weblogic-domain-pv-pvc/README.md]: pre-requisites > prerequisites

File: kubernetes/samples/scripts/create-weblogic-domain-pv-pvc/README.md [hunk 2]
```diff
@@ -0,0 +1,227 @@
...
+* Create a Kubernetes namespace for the persistent volume claim unless the intention is to use the default namespace.
+* Make sure that the host directory that will be used as the persistent volume must already exist and have the appropriate file permissions set.
```
Comment (rosemarymarano) [inline kubernetes/samples/scripts/create-weblogic-domain-pv-pvc/README.md]: Make sure that the host directory that will be used as the persistent volume already exists and has the appropriate file permissions set.

File: kubernetes/samples/scripts/create-weblogic-domain-pv-pvc/README.md [hunk 3]
```diff
@@ -0,0 +1,227 @@
...
+
+## Using sample scripts to create PV and PVC  
+
+Prior to run the `create-pv-pvc.sh` script, make a copy of the `create-pv-pvc-inputs.yaml` file, and uncommented and explicitly configure the inputs property `weblogicDomainStoragePath` in the inputs file.
```
Comment (rosemarymarano) [inline kubernetes/samples/scripts/create-weblogic-domain-pv-pvc/README.md]: Prior to running the `create-pv-pvc.sh` script, make a copy of the `create-pv-pvc-inputs.yaml` file, and uncomment and explicitly configure the `weblogicDomainStoragePath` property in the inputs file.

File: kubernetes/samples/scripts/create-weblogic-domain-pv-pvc/README.md [hunk 4]
```diff
@@ -0,0 +1,227 @@
...
+```
+$ sh create-pv-pvc.sh -h
+usage: create-pv-pvc.sh -i file -o dir [-e] [-h]
+  -i Parameter inputs file, must be specified.
+  -o Output directory for the generated yaml files, must be specified.
+  -e Also create the Kubernetes objects using the generated yaml files
+  -h Help
+```
+
+If you copy the sample scripts to a different location, make sure that you copy everything in the `<weblogic-kubernetes-operator-project>/kubernetes/samples/scripts` directory togather into the target directory, maintaining the orignal directory heirachy.
```
Comment (rosemarymarano) [inline kubernetes/samples/scripts/create-weblogic-domain-pv-pvc/README.md]: typo, togather > together

File: kubernetes/samples/scripts/create-weblogic-domain-pv-pvc/README.md [hunk 5]
```diff
@@ -0,0 +1,227 @@
...
+| `weblogicDomainStorageReclaimPolicy` | Kubernetes PVC policy for the persistent storage. The valid values are: 'Retain', 'Delete', and 'Recycle' | `Retain` |
+| `weblogicDomainStorageSize` | Total storage allocated for the PVC. | `10Gi` |
+| `weblogicDomainStorageType` | Type of storage. Legal values are `NFS` and `HOST_PATH`. If using 'NFS', weblogicDomainStorageNFSServer must be specified | `HOST_PATH` |
+| `weblogicDomainStorageNFSServer`| Name of the IP address of the NFS server. This setting only applies if weblogicDomainStorateType is NFS  | no default |
+
+By default, the domainUID is left empty in the inputs file so that the generated PV and PVC can be shared by multiple domain resources. For the use cases where dedicated PV and PVC are desired for a particular domain, the domainUID can be set, which will cause the generated PV and PVC associated with the domainUID specified. In the per domain PV and PVC case, the names of the generated yaml files and the Kubernetes PV and PVC objects are all decorated with the `domainUID`, and the PV and PVC obects are also labeled with the `domainUID`.
```
Comment (rosemarymarano) [inline kubernetes/samples/scripts/create-weblogic-domain-pv-pvc/README.md]: "..., which will cause the generated PV and PVC associated with the domainUID (to be?) specified."

**Comment by vanajamukkara** [comment]
Do you have a jenkins run with these changes?

**Comment by doxiao** [comment]
The Java integration test with this branch has one failure in Jenkins. I marked this PR as WIP while I am looking into the failure.

**Comment by doxiao** [comment]
Java integration test passed in Jenkins. http://wls-jenkins.us.oracle.com/job/weblogic-kubernetes-operator-javatest/240/.

**Comment by vanajamukkara** [comment]
I have a PR which got merged this afternoon - https://github.com/oracle/weblogic-kubernetes-operator/pull/465. You may want to get the latest from develop just in case.

**Pull request merged**